.vim/ftplugin/css.vim

```diff
@@ -0,0 +1,6 @@
+set shiftwidth=2
+set tabstop=2
+set softtabstop=2
+set expandtab
+
+set cc=
```

Comment:
But this is for css file. Look:
https://google.github.io/styleguide/htmlcssguide.xml?showone=Indentation#Indentation